Cargo.toml

```diff
@@ -212,12 +213,19 @@ x86_64 = { version = "0.14.9", default-features = false }
 xsave = { version = "2.0.2", default-features = false }
 zeroize = { version = "1.5.4", features = ["alloc"], default-features = false }
 
+# VFS
+wasmtime-vfs-dir = { version = "0.1.0", default-features = false }
+wasmtime-vfs-file = { version = "0.1.0", default-features = false }
+wasmtime-vfs-keyfs = { version = "0.1.1", default-features = false }
+wasmtime-vfs-ledger = { version = "0.1.0", default-features = false }
+wasmtime-vfs-memory = { version = "0.1.0", default-features = false }
+
 # wasmtime and its pinned dependencies
 # these will need to be updated together
-wasmtime = { git = "https://github.com/bytecodealliance/wasmtime", rev = "6f50ddaaf2ab8205b6e850361dd2cc662819f431", version = "2.0.0", features = ["cranelift", "pooling-allocator"], default-features = false }
+wasmtime = { version = "2.0.0", features = ["cranelift", "pooling-allocator"], default-features = false }
 cap-std = { version = "0.26.0", default-features = false }
 io-lifetimes = { version = "0.7.3", default-features = false }
-rustix = { version = "0.35.10", features = ["std"], default-features = false }
-wasi-common = { git = "https://github.com/bytecodealliance/wasmtime", rev = "6f50ddaaf2ab8205b6e850361dd2cc662819f431", version = "2.0.0", default-features = false }
-wasmtime-wasi = { git = "https://github.com/bytecodealliance/wasmtime", rev = "6f50ddaaf2ab8205b6e850361dd2cc662819f431", version = "2.0.0", features = ["sync"], default-features = false }
-wiggle = { git = "https://github.com/bytecodealliance/wasmtime", rev = "6f50ddaaf2ab8205b6e850361dd2cc662819f431", version = "2.0.0", default-features = false }
```

Comment:
Why don't we go full 3.0.1 here?

Reply:
VFS uses 2.x https://github.com/enarx/vfs/blob/a399de095c8b65aa5579795fc58a498dc6f38a30/Cargo.toml#L60